scripts/multimodal/run_multimodal.py

```diff
@@ -0,0 +1,60 @@
+import os
+from pathlib import Path
+
+os.chdir(Path(__file__).parent.parent)
```

Comment:
would probably not change the directory, but probably just do:

```suggestion
repo_path = Path(__file__).parent.parent
```

and use that to construct your out_folder

Reply:
There is a reason for doing this. I want the script to import code from the repo not from the package, and if you are in the root directory of the repo running:
```bash
python3 scripts/multimodal/run_benchmark.py
```
it will fail to import stuff

Reply:
But shouldn't it be the same? Otherwise I think that is a problem

Also if that is the case `sys.path.append` is your way to do that without changing your directory